### ISSUE TITLE
add docs for gradient clipping

### ISSUE THREAD

**Issue by CarloLucibello** [CarloLucibello/Tsunami.jl#39]
https://lightning.ai/docs/pytorch/stable/advanced/training_tricks.html#gradient-clipping

**Comment by CarloLucibello** [comment]
We don't actually have to expose a trainer option for this since gradient clipping can be inserted into an OptimiseresChains.

https://fluxml.ai/Optimisers.jl/dev/api/#Optimisers.ClipGrad

**Comment by CarloLucibello** [comment]
fixed in #57
